### ISSUE TITLE
[WEB] Unsupported operation: Platform._version

### ISSUE THREAD

**Issue by BartusZak** [CodingAleCR/http_interceptor#53]
```dart
HttpClientWithInterceptor _httpClient;
...
await _httpClient.get();
```

Result:
`Unsupported operation: Platform._version`

```
Flutter 1.26.0-17.3.pre • channel beta • https://github.com/flutter/flutter.git
Framework • revision 4b50ca7f7f (8 days ago) • 2021-02-04 19:44:27 -0800       
Engine • revision 2c527d6c7e
Tools • Dart 2.12.0 (build 2.12.0-259.8.beta)
```

`http_interceptor: ^0.3.3`

Related to https://github.com/flutter/flutter/issues/39998.

`<getObject: NoSuchMethodError: The getter 'uri' was called on null.>`
`http_client_with_interceptor.dart`

**Comment by BartusZak** [comment]
#https://github.com/flutter/flutter/issues/39998

**Comment by CodingAleCR** [comment]
Are you by any chance using Flutter for Web?

**Comment by BartusZak** [comment]
Hey @CodingAleCR,

That's correct.

I was playing around with your library but at the end I had to user `Dio`. 

It has `interceptors` and supports `Web`/`Android` and `iOS`.

**Comment by CodingAleCR** [comment]
I'm sorry for the confusion, I have not been able to take the time to add support for Web. I'm glad you found a solution for your problems and happy that you are using Flutter. Cheers from Costa Rica my friend! 🥳🥳

**Comment by BartusZak** [comment]
Let's leave that issue open then for others.

Pozdrowienia z Polski 👯‍♂️

**Comment by CodingAleCR** [comment]
This should be fixed as of 0.4.0, it's important that you guys know that the badCertificatesCallback and the findProxy features are not supported on Flutter Web due to browser limitations. Let me know if any other issues persist, also be aware of CORS as while testing I found some issues around it while querying public APIs.
